frontend/app/templates/work_packages/tabs/activity.html

```diff
@@ -10,12 +10,14 @@
           <h3 class="activity-date"
               ng-if="currentDate !== (activities[$index-1].props.createdAt | date:'longDate')"
               ng-bind="currentDate"/>
-          <user-activity work-package="workPackage"
-                         activity="activity"
-                         activity-no="activityNo"
-                         is-initial="isInitial"
-                         input-element-id="'add-comment-text'">
-          </user-activity>
+            <activity-entry work-package="workPackage"
+                           activity="activity"
+                           activity-no="activityNo"
+                           is-initial="isInitial"
```

Comment:
That's a good point, and something I'd have to check with how the journal is initially filled when a work package is created. 

I don't think the second one would work, though, or can you mention a work package prior to its creation using the `#id` notation? Even if you mention some WP unintentionally, the created journal entry still would precede it.